### PR TITLE
Nav bar component

### PR DESCRIPTION
**Pull request by Maarimar**
finish styling

### CODE REVIEW

**Review by Count-MonteCristo — APPROVED** [review]

File: src/components/pages/map.js
Comment (dkprioleau) [inline src/components/pages/map.js:1]: Remove comments

**Review by dkprioleau — CHANGES_REQUESTED** [review]
A lot of the files you made changes to (lost and found files & icon images) you  made in another PR so you don't need to make them again. They will be added once your other PR is approved.  You're two other PR's need to be merged first since there seems to be an error from the icon files.

**Review by dkprioleau — APPROVED** [review]
LGTM, I also noticed the error has disappeared so you can go ahead and merge. I noticed not all the pages have the navbar displaying though: editProfile, selectLorOrFound, addLostPet, addFoundPet. You can either add another commit to this PR or make another issue to add navbar to these pages

**Comment by Maarimar** [comment]
> LGTM, I also noticed the error has disappeared so you can go ahead and merge. I noticed not all the pages have the navbar displaying though: editProfile, selectLorOrFound, addLostPet, addFoundPet. You can either add another commit to this PR or make another issue to add navbar to these pages

I followed the design we have on Figma :) we can discuss if the navbar goes everywhere during ore next meeting

**Pull request merged**